Character/Classes/Fighter/Fighter (Cavalier).xml

```diff
@@ -1,154 +1,63 @@
-	<text />
-	<text>Add roll to damage, knock prone (Large) vs Strength Saving Throw</text>
-	<text>At 7th (class) level, use twice for disadvantage.</text>
-	<roll>1d8</roll>
-	<roll>1d10</roll>
-	<roll>1d12</roll>
-  </spell>
-  <spell PS="PS" UA="Revised Class Options">
-	<name>Warding Maneuver</name>
-	<level>1</level>
-	<school/>
-	<time>1 reaction</time>
-	<range>creature within 5 feet</range>
-	<components />
-	<duration />
-	<classes>Fighter (Cavalier)</classes>
-	<text>If you or a creature within 5 feet of you is hit by an attack, you can expend one superiority die as a reaction if you're wielding a weapon or a shield. Roll the die, and add the number rolled to the target's AC against that attack. If the attack still hits, the target has resistance against the attack's damage.</text>
-	<text/>
-	<text>Add roll to AC, resistance if still hit.</text>
-	<roll>1d8</roll>
-	<roll>1d10</roll>
-	<roll>1d12</roll>
-  </spell>
-
-	<spellList class="Fighter (Cavalier)" UA="Revised Class Options">
-		<!-- Cavalier currently has access to PHB Battle Master Maneuvers 'Precision Attack' and 'Trip Attack',
-			with minor text changes. for now, they are duplicated with additional FC5 meta data vs Battle Master -->
-		<!--<index name="Precision Attack" />-->
-		<!--<index name="Trip Attack" />-->
-	</spellList>
-
-</compendium>
\ No newline at end of file
+	<subclass baseclass="Fighter">
+		<name>Fighter (Cavalier)</name>
+		<spellAbility>Strength</spellAbility>
+
+		<autolevel level="3">
+			<feature optional="YES">
+				<name>Martial Archetype: Cavalier</name>
+				<text>The archetypal cavalier excels at mounted combat. Usually born among the nobility and raised at court, a cavalier is equally at home leading a cavalry charge or exchanging repartee at a state dinner. Cavaliers also learn how to guard those in their charge from harm, often serving as the protectors of their superiors and of the weak. Compelled to right wrongs or earn prestige, many of these fighters leave their lives of comfort to embark on glorious adventure.</text>
+			</feature>
+
+			<feature optional="YES">
+				<name>Cavalier: Bonus Proficiency</name>
+				<text>When you choose this archetype at 3rd level, you gain proficiency in one of the following skills of your choice: Animal Handling, History, Insight, Performance, or Persuasion. Alternatively, you learn one language of your choice.</text>
+			</feature>
+			<feature optional="YES">
+				<name>Cavalier: Born to the Saddle</name>
+				<text>Starting at 3rd level, your mastery as a rider becomes apparent. You have advantage on saving throws made to avoid falling off your mount. If you fall off your mount and descend no more than 10 feet, you can land on your feet if you’re not incapacitated.</text>
```

Comment:
```suggestion
				<text>Starting at 3rd level, your mastery as a rider becomes apparent. You have advantage on saving throws made to avoid falling off your mount. If you fall off your mount and descend no more than 10 feet, you can land on your feet if you're not incapacitated.</text>
```